syntax = "proto2";
package y_data;

message vector {
    repeated double vec_value = 2;
}

message full_y {
    repeated vector vec_t = 1;
}